lighthouse-cli/sentry-prompt.ts

```diff
@@ -0,0 +1,67 @@
+import {existsSync} from 'fs';
+import {join as joinPath} from 'path';
+import {inquirer, Configstore} from './shim-modules';
+const log = require('../lighthouse-core/lib/log');
+
+const MAXIMUM_WAIT_TIME = 20 * 1000;
+
+const MESSAGE = [
+  'Lighthouse would like to report back any errors that might occur while auditing. \n  ',
+  'Information such as the URL you are auditing, its subresources, your operating system, Chrome, ',
+  'and Lighthouse versions may be recorded. Would you be willing to have Lighthouse automatically ',
+  'report this information to the team to aid in improving the product?',
+].join('');
+
+async function prompt() {
+  if (!process.stdout.isTTY || process.env.CI) {
+    // Default non-interactive sessions to false
+    return false;
+  }
+
+  let timeout: NodeJS.Timer;
+
+  const prompt = inquirer.prompt([
```

Comment:
getting an exception in the prompt() method:

![image](https://user-images.githubusercontent.com/39191/27300175-a5e8965a-54e3-11e7-8a56-65963fe44aaa.png)